examples/hooks/App.tsx

```diff
@@ -81,6 +82,17 @@ export function App() {
               <RangeInput attribute="price" />
             </Panel>
           </DynamicWidgets>
+          <Panel header="NumericMenu">
```

Comment:
Could this panel also be handled by `<DynamicWidgets>` for consistency?

Also, I see some panel names refering to the type of widgets, and other refering to the faceting attribute, maybe we could choose one or the other but not mix it, wdyt?

Reply:
The `<NumericMenu>` doesn't display when within `<DynamicWidgets>`. I believe that's because of the Algolia index configuration.

Can you confirm @Haroenv?

Reply:
The inconsistency started with the "Hierarchy" panel because the `categories` and `hierarchicalCategories` are essentially the same. We should find another name though.

I'll rename this panel to "Price range" if that's good for you.

Reply:
See db6e4e1a8e93781e8d33b9e2a32c0b6326417344.